examples/language-modeling/README.md

```diff
@@ -321,7 +321,8 @@ python run_clm.py \
 
 ## PEFT
 
-To run LoRA finetuning and inference. you could use `run_lora_clm.py` as an example. Multi-card examples can be simply adapted to run LoRA finetuning. Here is the CLM example with Llama1-7B and Falcon-40B:
+To run LoRA finetuning, you can use `run_lora_clm.py`. Multi-card examples can be simply adapted to run LoRA finetuning.
+Here are command examples for Llama1-7B, Falcon-40B and Llama2-70B:
```

Comment:
```suggestion
To run LoRA finetuning, you can use `run_lora_clm.py`.
Here are single-/multi-device command examples for Llama1-7B, Falcon-40B and Llama2-70B:
```
Let's change that too, it sounds a bit redundant.

Reply:
Agree. addressed